documentation/source/simulator_support.rst

```diff
@@ -115,25 +140,44 @@ Aldec Riviera-PRO
 The :envvar:`LICENSE_QUEUE` environment variable can be used for this simulator –
 this setting will be mirrored in the TCL ``license_queue`` variable to control runtime license checkouts.
 
+
 .. _sim-activehdl:
 
 Aldec Active-HDL
 ================
 
+In order to use this simulator, set :make:var:`SIM` to ``activehdl``:
+
+.. code-block:: bash
+
+    make SIM=activehdl
+
 
 .. _sim-questa:
 
 Mentor Questa
 =============
 
-See :ref:`sim-modelsim`.
+In order to use this simulator, set :make:var:`SIM` to ``questa``:
+
+.. code-block:: bash
+
+    make SIM=questa
+
+For more information, see :ref:`sim-modelsim`.
 
 
 .. _sim-modelsim:
 
 Mentor ModelSim
 ===============
 
+In order to use this simulator, set :make:var:`SIM` to ``modelsim``:
+
+.. code-block:: bash
+
+    make SIM=modelsim
```

Comment:
I am not sure if we should not do `Mentor Questa/Modelsim` since this is one and the same at this point?

Reply:
Maybe... But not in this PR :)